lib/vlc.js

```diff
@@ -173,12 +268,23 @@ module.exports = (function() {
   }
 
   var volume = function(change) {
-    if(change == '0') {
+    if(json.env_OS=="linux")
+    {
+      change=change +"%";
+      if(change == '0') {
+      var child = spawn("pactl",[ "--","set-sink-volume","0","0%"] );
+          }
+    else {
+      var child = spawn("pactl",[ "--","set-sink-volume","0",change] );
+    }
+    }
+    else{if(change == '0') {
       var child = spawn("nircmd.exe", ["mutesysvolume", "2"]);
     }
     else {
-      var child = spawn("nircmd.exe", ["changesysvolume", change]);
-    }
+      change=change*65535/100;
+      var child = spawn("nircmd.exe", ["setsysvolume", change]);
```

Comment:
I feel because you are using so many spawn commands now, it would be better to separate them out in a file. Let's say the name of file is `commands.js`:

```js
var commands = {
   playAndExit: {
    command: "vlc",
    args: ["--play-and-exit", ".."]
   },
   nextCommand: {
   //
   }
};

function runCommand(command, args) {
   if (!args) {
       args = [];
   }
   args.concat(commands[command].args);
   spawn(commands[command].command, args);
}

runCommand('playAndExit', ["any_extra_commands_here"]);
```

This is kind of barebone code you can implement.